src/components/ContextMenuItem.js

```diff
@@ -89,6 +99,7 @@ function ContextMenuItem({onPress, successIcon, successText, icon, text, isMini,
             descriptionTextStyle={styles.breakAll}
             style={getContextMenuItemStyles(windowWidth)}
             isAnonymousAction={isAnonymousAction}
+            focused={focused}
```

Comment:
I also thought `isFocused` name is better but didn't request change because `focused` prop name already exists here 😄

Reply:
I think that's fine as we didn't touch `MenuItem` component here.
If anyone is adding/changing props on that component in the future, they might be responsible.
cc: @francoisl